fulp_modules/features/antagonists/infiltrators/infil_items.dm

```diff
@@ -224,29 +224,34 @@
 	if(!criminal)
 		to_chat(user, span_notice("You don't understand how this injector works."))
 		return
+	var/mob/living/carbon/human/man = victim
```

Comment:
why are we doing this before the ishuman check ?

Reply:
accident